ios/Video/RCTVideo.swift

```diff
@@ -63,6 +63,7 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
     private var _presentingViewController: UIViewController?
     private var _startPosition: Float64 = -1
     private var _showNotificationControls = false
+    private var _lastBitrate = -2.0
```

Comment:
I think it's better to set it to `-1.0` like `_startPosition` because a new developer working on the library might be confused if it's set to `-2.0`.
WDYT?

Reply:
Yes, In fact this is intended, but I will add comment. Apperently, AVPlayer sometime report -1 (not sure why), If I initilize the value to -1, this value will be missed !